### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Devanshi-Sharma12**
This is the testing branch of my updated_t20_list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# updated_t20_list
+Welcome to my project!!!
```
Comment (pandeytapan) [inline README.md]: Put a proper project name

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# updated_t20_list
+Welcome to my project
 This project contains a Python script to find and display player statistics from a CSV file containing data about the top 5 country players in the recent T20 World Cup.The data is collected from ICC official site.
```
Comment (pandeytapan) [inline README.md:4]: Write all the code elements in proper back-ticks with examples showing the answers
Use unordered/ ordered lists wherever applicable.

Comment (pandeytapan) [reply README.md:4]: Read the following [Markdown guide](https://www.markdownguide.org/)

**Review by pandeytapan — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,29 +1,65 @@
...
+       ```
+
+
+3. Expected Output:
 The script will output the statistics for Jasprit Bumrah:
-Stats for Jasprit Bumrah:
-Player: Jasprit Bumrah
-Country: India
-Wickets: 15
-Average: 8.26
```
Comment (pandeytapan) [inline README.md]: The output must be in code fences with type e.g.
```bash
output
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,65 @@
...
 
+    Stats for Jasprit Bumrah:
+    <html>
```
Comment (pandeytapan) [inline README.md]: Why we have HTML tags around the output?

**Review by pandeytapan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Devanshi-Sharma12) [reply README.md]: written sir

File: README.md (hunk 2 above)
Comment (Devanshi-Sharma12) [reply README.md:4]: done sir. both the things are resolved

File: README.md (hunk 3 above)
Comment (Devanshi-Sharma12) [reply README.md]: Both the things are done sir

File: README.md (hunk 4 above)
Comment (Devanshi-Sharma12) [reply README.md]: yes i have removed them

**Pull request merged**